pkg/diagnostics/diagnostic_bundle_test.go

```diff
@@ -450,12 +450,24 @@ func machineConfigs() []providers.MachineConfig {
 func TestGenerateManagementClusterBundle(t *testing.T) {
 	kubeconfig := "testcluster.kubeconfig"
 
-	spec := test.NewClusterSpec(func(s *cluster.Spec) {
+	bs_spec := test.NewClusterSpec(func(s *cluster.Spec) {
```

Comment:
nit: can we use camelCase
```suggestion
	bootstrapSpec := test.NewClusterSpec(func(s *cluster.Spec) {
```

Reply:
Sure. Thanks!